docs/python-reference/dtype.ipynb

```diff
@@ -82,8 +91,163 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Datetime Variables always need a temporal unit and that unit determines how the integer that is passed to `value=` is interpreted:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "var = sc.scalar(value=681794055, unit=sc.units.ns, dtype=sc.dtype.datetime64)\n",
+    "var"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Datetime elements are automatically converted to and from [numpy.datetime64](https://numpy.org/doc/stable/reference/arrays.datetime.html#basic-datetimes) objects:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "var.value"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "now = sc.scalar(value=np.datetime64('now'))\n",
+    "now"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that `now` has unit `s` even though we did not specify it.\n",
+    "The unit was deduced from the `numpy.datetime64` object which encodes a unit of its own."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Operations\n",
+    "\n",
+    "Variables containing datetimes only support a limited set of operations as it makes no sense to, for instance, add two time points.\n",
+    "In contrast to numpy, scipp does not have a separate type for time differences.\n",
+    "Those are simply encoded by integer Variables with a temporal unit."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a = sc.scalar(value=np.datetime64('2021-03-14', 'ms'))\n",
+    "b = sc.scalar(value=np.datetime64('2000-01-01', 'ms'))\n",
+    "a - b"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    a + b\n",
+    "except sc.DTypeError as err:\n",
+    "    print(err)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a + sc.scalar(value=123, unit='ms')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Time zones\n",
+    "\n",
+    "Scipp does not support manual handling of time zones.\n",
+    "All datetime objects are assumed to be in UTC.\n",
+    "Scipp does not look at your local time zone, thus the following will always produce 12:30 on 2021-03-09 UTC no matter where you are when you run this code:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sc.scalar(value=np.datetime64('2021-09-03T12:30:00'))"
+   ]
```

Comment:
Is it worth making a separate page for datetime?

Reply:
Hm, maybe. It is quite short at the moment